templates/home/home-en.html

```diff
@@ -44,90 +39,71 @@ <h2 class="mrgn-tp-md">Most requested</h2>
-		</div>
-
-		<div class="col-lg-4 col-md-6">
-			<h3><a href="https://www.canada.ca/en/services/business.html">Business and industry</a></h3>
-			<p>Starting a business, permits, copyright, business support, selling to government</p>
-		</div>
-		<div class="col-lg-4 col-md-6">
-			<h3><a href="https://www.canada.ca/en/services/benefits.html">Benefits</a></h3>
-			<p>EI, family and sickness leave, pensions, housing, student aid, disabilities</p>
-		</div>
-		<div class="col-lg-4 col-md-6">
-			<h3><a href="https://www.canada.ca/en/services/health.html">Health</a></h3>
-			<p>Food, nutrition, diseases, vaccines, drugs, product safety and recalls</p>
-		</div>
-		<div class="col-lg-4 col-md-6">
-			<h3><a href="https://www.canada.ca/en/services/taxes.html">Taxes</a></h3>
-			<p>Income tax, payroll, GST/HST,  contribution limits, tax credits, charities</p>
-		</div>
-		<div class="col-lg-4 col-md-6">
-			<h3><a href="https://www.canada.ca/en/services/environment.html">Environment and natural resources</a></h3>
-			<p>Weather, climate, agriculture, wildlife, pollution, conservation, fisheries</p>
-		</div>
-		<div class="col-lg-4 col-md-6">
-			<h3><a href="https://www.canada.ca/en/services/defence.html">National security and defence</a></h3>
-			<p>Military, transportation and cyber security, securing the border, counter-terrorism</p>
-		</div>
-		<div class="col-lg-4 col-md-6">
-			<h3><a href="https://www.canada.ca/en/services/culture.html">Culture, history and sport</a></h3>
-			<p>Arts, media, heritage, official languages, national identity and funding</p>
-		</div>
-		<div class="col-lg-4 col-md-6">
-			<h3><a href="https://www.canada.ca/en/services/policing.html">Policing, justice and emergencies</a></h3>
-			<p>Safety, justice system, prepare for emergencies, services for victims of crime</p>
-		</div>
-		<div class="col-lg-4 col-md-6">
-			<h3><a href="https://www.canada.ca/en/services/transport.html">Transport and infrastructure</a></h3>
-			<p>Aviation, marine, road, rail, dangerous goods, infrastructure projects</p>
-		</div>
-		<div class="col-lg-4 col-md-6">
-			<h3><a href="https://international.gc.ca/world-monde/index.aspx?lang=eng">Canada and the world</a></h3>
-			<p>Foreign policy, trade agreements, development work, global issues</p>
-		</div>
-		<div class="col-lg-4 col-md-6">
-			<h3><a href="https://www.canada.ca/en/services/finance.html">Money and finances</a></h3>
-			<p>Personal finance, credit reports, fraud protection, paying for education</p>
-		</div>
-		<div class="col-lg-4 col-md-6">
-			<h3><a href="https://www.canada.ca/en/services/science.html">Science and innovation</a></h3>
-			<p>Scientific research on health, environment and space, grants and funding</p>
-		</div>
-	</div>
-</section>
+<section class="gc-srvinfo">
+	<div class="container">
+		<h2 class="wb-inv">Services and information</h2>
+		<div class="wb-eqht row">
```

Comment:
Are we not using the equal height CSS now? Well I see that we didn't updated that pattern to leverage the equal height CSS now. May be something to consider in the future. No need to make any adjustment.